### DISCUSSION TITLE
[Question][Offline Lecture] Will there be an offline lecture tomorrow?

### DISCUSSION THREAD

**Discussion by Allgot** [prosyslab-classroom/cs424-program-reasoning#380] [Q&A]
Hello.
I'm just confusing when the online lecture will turn into the offline classes.
Especially, I'm wondering whether there will be an offline class tomorrow.. because the incoming typhoon is forecasted to affect Daejeon from today late-night to tomorrow morning accroding to KMA.

![image](https://user-images.githubusercontent.com/96973603/188426774-065ebc7a-eac0-4f31-9176-f1c9151e5152.png)

Thank you for replying in advance!

**Comment by KihongHeo** [comment]
OK. Let us go online tomorrow. TAs will send a notification.
